run/test/specs/message_checks_groups_ios.spec.ts

```diff
@@ -0,0 +1,507 @@
+  await device2.longPressMessage(testMessage);
+  await device2.clickOnElement("Reply to message");
+  await device2.sendMessage(replyMessage);
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  // Close server and devices
+  await closeApp(device1, device2, device3);
+}
+
+async function sendVoiceMessageGroup(platform: SupportedPlatformsType) {
+  const testGroupName = "Message checks for groups";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
+
+  // Create contact between User A and User B
+  await createGroup(
+    platform,
+    device1,
+    userA,
+    device2,
+    userB,
+    device3,
+    userC,
+    testGroupName
+  );
+  const replyMessage = `Replying to voice message from ${userA.userName} in ${testGroupName}`;
+  await device1.longPress("New voice message");
+  // "Session" would like to access the microphone (Don't allow/ OK)
+  await device1.clickOnElement("OK");
+  await device1.pressAndHold("New voice message");
+
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Voice message",
+  });
+  await device2.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Voice message",
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Voice message",
+  });
+
+  await device2.longPress("Voice message");
+  await device2.clickOnElement("Reply to message");
+  await device2.sendMessage(replyMessage);
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  // Close server and devices
+  await closeApp(device1, device2, device3);
+}
+
+// async function sendDocGroup(platform: SupportedPlatformsType) {
+//   const testGroupName = "Message checks for groups";
+//   const { device1, device2, device3 } = await openAppThreeDevices(platform);
+//   // Create users A, B and C
+//   const [userA, userB, userC] = await Promise.all([
+//     newUser(device1, "Alice", platform),
+//     newUser(device2, "Bob", platform),
+//     newUser(device3, "Charlie", platform),
+//   ]);
+
+//   // Create contact between User A and User B
+//   await createGroup(
+//     platform,
+//     device1,
+//     userA,
+//     device2,
+//     userB,
+//     device3,
+//     userC,
+//     testGroupName
+//   );
+// }
```

Comment:
maybe uncomment it and make it .skip() so we know it is still to be done?

Reply:
Can't do skip on individual tests but just on the describe part